UnleashClient/api/register.py

```diff
@@ -41,22 +43,26 @@ def register_client(
     :param request_timeout:
     :return: true if registration successful, false if registration unsuccessful or exception.
     """
-    registation_request = {
+    registration_request = {
```

Comment:
Not related to this PR but the spelling is driving me mad and I was in the neighborhood